### PR TITLE
Bug 911664 - [bb][refactor] Tabs and filters

### PR DESCRIPTION
**Pull request by basiclines**
- Removed layout positions for `bottom` cases, just added the classname for changing background orientation
- Created `skin-filter` for filter component
- Refactorized selectors
- More responsive layout
- Added landscape support for filters
- Removed `:target` interaction, has it has been proving to be creating problems

### CODE REVIEW

File: style/tabs/skin.default.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+.bb-tabs li[aria-selected] a { /* Selected state */
+  color: #62c6f5
+}
+
+.bb-tabs li[aria-selected] .icon:after { /* Selected state icon */
+  background-position: center -4.5rem;
+}
+
+.bb-tabs li[aria-disabled] a { /* Disabled state */
+  color: #4d4d4d;
```
Comment (rnowm) [inline style/tabs/skin.default.css]: correct color based on Common Controls doc is #a1a1a2

File: style/tabs/skin.default.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  background: #000;
+}
+
+.bb-tabs a:active { /* Pressed state */
+  background-color: #008aaa;
+  color: #fff;
+}
+
+.bb-tabs li[aria-selected] a { /* Selected state */
+  color: #62c6f5
```
Comment (rnowm) [inline style/tabs/skin.default.css]: correct color based on Common Controls doc is #00caf2

File: style/tabs/skin.default.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+.bb-tabs a {
+  font-size: 1.3rem;
+  color: #fff;
+  background: #000;
+}
+
+.bb-tabs a:active { /* Pressed state */
+  background-color: #008aaa;
```
Comment (rnowm) [inline style/tabs/skin.default.css]: Pressed state should be: #008caa

File: README.md [hunk 4]
```diff
@@ -117,6 +117,7 @@ Class names index
 * `.bbs-organic`: headers.css skin (widely used in settings views)
 * `.bbs-dark`: headers.css skin (used for media apps)
 * `.bbs-flat`: buttons.css skin (commonly applied to buttons lists)
+* `.bbs-filter`: tabs.css skin (used for filtering content)
 
 * `.icon-add`: headers.css icon
 * `.icon-compose`: headers.css icon
```
Comment (rnowm) [inline README.md:123]: add `bottom`class description used in filters

**Comment by basiclines** [comment]
Everything fixed thanks @rnowm for the help in using common controls real colors!

**Pull request merged**